src/portals/hope/features/claims/claim-details/ClaimCounterparty/CounterpartyDetails.tsx

```diff
@@ -0,0 +1,186 @@
+import styled from '@emotion/styled'
+import { Button, convertEnumToTitle, Input, Select, Spacing } from '@hedvig-ui'
+import gql from 'graphql-tag'
+import { InvoiceModal } from 'portals/hope/features/claims/claim-details/ClaimCounterparty/InvoiceModal'
+import { LiabilityLevel } from 'portals/hope/features/config/constants'
+import React, { useState } from 'react'
+import { toast } from 'react-hot-toast'
+import {
+  ClaimCounterpartyInfoFragment,
+  RecoveryInvoice,
+  useCreateCounterpartyRecoveryInvoiceMutation,
+  useInsuranceProvidersQuery,
+  useUpdateCounterpartyInsuranceProviderMutation,
+  useUpdateCounterpartyLiabilityLevelMutation,
+} from 'types/generated/graphql'
+
+gql`
+  query InsuranceProviders {
+    insuranceProviders {
+      id
+    }
+  }
```

Comment:
We have more on the Insurance Providers now, unsure exactly how to use it though 🤔 
```suggestion
  query InsuranceProviders {
    insuranceProviders {
      id
      displayName
      email
      phone
    }
  }
```